pallets/proxy-wrapper/Cargo.toml

```diff
@@ -0,0 +1,51 @@
+[package]
+name = "pallet-proxy-wrapper"
```

Comment:
@TarekkMA Alex approved name `free-proxy`, so lets rename and I try to push Alex approve PR